scipy/misc/__init__.py

```diff
@@ -32,13 +28,25 @@
    imshow - Simple showing of an image through an external viewer [requires Pillow]
    info - Get help information for a function, class, or module
    lena - Get classic image processing example image Lena
+   toimage - Takes a numpy array and returns a PIL image [requires Pillow]
+   source - Print function source code
```

Comment:
Why are these moved out of the deprecated functions list? They should be used as numpy functions, not scipy.misc aliases